packages/components/inputs/rich-text-utils/src/rich-text-body/rich-text-body.styles.ts

```diff
@@ -28,17 +35,18 @@ const getBorderColor = (props) => {
   `;
 };
 
-const getBackgroundColor = (props) => {
+const getBackgroundColor = (props: Props) => {
   if (props.isDisabled) {
     return css`
       background-color: ${vars.backgroundColorForInputWhenDisabled};
     `;
   }
-  if (props.isReadOnly) {
-    return css`
-      background-color: ${vars.backgroundColorForInputWhenReadonly};
-    `;
-  }
+  //Question: imported vars has no property with `backgroundColorForInputWhenReadonly` as key. Should this be defined?
```

Comment:
Did you forget to uncomment this?

Reply:
Yes I forgot. lol, thank you. Will clean this up.